templates/base.html

```diff
@@ -4,6 +4,8 @@
 <head>
     <title>{% block title %}User Management{% endblock %}</title>
     <link rel="stylesheet" href="{% static 'css/bootstrap.min.css' %}">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
```

Comment:
What is the purpose of this file?

Reply:
> What is the purpose of this file?

To load this content on each template. the cross on the messages that will make it disappear on clicking was not working without js that why i added it ,we can remove the cross on the messages if we want to remove js.